README.md

```diff
@@ -0,0 +1,18 @@
+Python Tic-Tac-Toe
+==================
+
+### Getting Started
+
+Run `python3 start.py` in the __PythonTTT__ folder.
+
+### Playing the game
```

Comment:
I'd also like to see a section on how to run tests and coverage.

Reply:
I'll add a section the explains coverage run -m pytest, as well as anything the user may need to install to be able to run them.